### PR TITLE
Update contributors section in README.md

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,5 +20,16 @@ Read about future plans in [our roadmap](https://docs.google.com/document/d/1fsc
 
 ## Team
 
-The development of Vega-Lite is led by [Kanit "Ham" Wongsuphasawat](https://twitter.com/kanitw), [Dominik Moritz](https://twitter.com/domoritz), [Arvind Satyanarayan](https://twitter.com/arvindsatya1), and [Jeffrey Heer](https://twitter.com/jeffrey_heer) of the [University Washington Interactive Data Lab](https://idl.cs.washington.edu), with significant help from [Will Strimling](https://willium.com), [Yuhan (Zoe) Lu](https://github.com/YuhanLu), [Matthew Chun](http://mattwchun.com/), and [Akshat Shrivastava](https://github.com/AkshatSh).
+The development of Vega-Lite is led by the alumni and members of the [University Washington Interactive Data Lab](https://idl.cs.washington.edu) (UW IDL), including [Kanit "Ham" Wongsuphasawat](https://twitter.com/kanitw) (now at Apple), [Dominik Moritz](https://twitter.com/domoritz) (UW IDL), [Arvind Satyanarayan](https://twitter.com/arvindsatya1) (now at MIT), and [Jeffrey Heer](https://twitter.com/jeffrey_heer) (UW IDL). 
+Many contributors including
```
Comment (domoritz) [inline README.md]: "Vega-Lite gets significant contributions from its community. In particular:"

I like highlighting the community aspects.

File: site/index.md [hunk 2]
```diff
@@ -79,4 +79,15 @@ Read our [introduction article to Vega-Lite v2 on Medium](https://medium.com/@uw
 
 ## Team
```
Comment (domoritz) [inline site/index.md]: What do you think about removing this and instead writing: 

Vega-Lite was started at the [Interactive Data Lab at the University of Washington] and now being developed by a [community of contributors](https://github.com/vega/vega-lite/graphs/contributors).

File: README.md [hunk 3]
```diff
@@ -20,5 +20,16 @@ Read about future plans in [our roadmap](https://docs.google.com/document/d/1fsc
 
 ## Team
 
-The development of Vega-Lite is led by [Kanit "Ham" Wongsuphasawat](https://twitter.com/kanitw), [Dominik Moritz](https://twitter.com/domoritz), [Arvind Satyanarayan](https://twitter.com/arvindsatya1), and [Jeffrey Heer](https://twitter.com/jeffrey_heer) of the [University Washington Interactive Data Lab](https://idl.cs.washington.edu), with significant help from [Will Strimling](https://willium.com), [Yuhan (Zoe) Lu](https://github.com/YuhanLu), [Matthew Chun](http://mattwchun.com/), and [Akshat Shrivastava](https://github.com/AkshatSh).
+The development of Vega-Lite is led by the alumni and members of the [University Washington Interactive Data Lab](https://idl.cs.washington.edu) (UW IDL), including [Kanit "Ham" Wongsuphasawat](https://twitter.com/kanitw) (now at Apple), [Dominik Moritz](https://twitter.com/domoritz) (UW IDL), [Arvind Satyanarayan](https://twitter.com/arvindsatya1) (now at MIT), and [Jeffrey Heer](https://twitter.com/jeffrey_heer) (UW IDL).
```
Comment (domoritz) [inline README.md]: University OF Washington IDL

File: site/index.md (hunk 2 above)
Comment (kanitw) [reply site/index.md]: I like the sentence better ,but probably still nice to list major contributors.

Comment (domoritz) [reply site/index.md]: We still list them on the readme and at the link.

**Pull request closed without merging**